test/user.js

```diff
@@ -0,0 +1,103 @@
+//During the test the env variable is set to test
+process.env.NODE_ENV = 'development';
+
+let chai = require('chai'),
+  chaiHttp = require('chai-http'),
+  server = require('../app'),
+  should = chai.should(),
+  dictum = require('dictum.js'),
+  User = require('../app/models').users;
+
+chai.use(chaiHttp);
+
+describe('User', () => {
+  beforeEach((done) => {
```

Comment:
This should not be necessary

Reply:
what should not be necessary? the use(chaiHttp), the describe('user') or the beforeEach()?